libparsec/crates/testbed/README.md

```diff
@@ -0,0 +1,70 @@
+# Libparsec Testbed
+
+The `libparsec_testbed` crate defines common organization templates to prevent code
+duplication in tests.
+
+## Organization templates
+
+A template defines the structure of an organization such as:
+
+- the users & their devices
+- the workspaces & their sharing rules
+- the files and directories inside each workspace
+
+Within a test, this is usually a good starting point (hence the name) and the
+organization can be further customized in a way that is relevant to the test
+(e.g. revoke an existing user).
+
+Templates also allow to create deterministic tests based on the events that
+should be triggered, the order in which they should be triggered or the
+specific timestamp at which they should be triggered.
+
+## Use in tests
+
+The templates defined in this crate are used to test Libparsec, Parsec CLI and
+the Parsec Server.
+
+### Libparsec testing
+
+The templates can be enabled via the `#[parsec_test]` macro defined in the
+`tests_macros` crate.
+
+```rust
+#[parsec_test(testbed = "coolorg")]
+async fn my_test_function(
+    // ...
+    env: &TestbedEnv,
+){
+  // access and customize the organization via the TestbedEnv
+}
+```
+
+### Parsec CLI testing
+
+TODO
+
+### Parsec Server testing
+
+The templates are exposed as pytest fixtures in `server/tests/common/client.py`.
+
+TODO: some tests access the TestbedBackend interface defined in CLI.
```

Comment:
Is that a temporary shortcut or is it the intended way? In any case it will be nice to explain it here.